packages/eslint-plugin/src/rules/enforce-module-boundaries.ts

```diff
@@ -358,14 +358,20 @@ export default createESLintRule<Options, MessageIds>({
                         dirname(fileName),
                         dirname(importPath)
                       );
+                      const maybeTagRelative = (path: string): string =>
```

Comment:
Why was this needed? Wasn't `relative(...)` above already returning a relative path?

Reply:
it's relative but when there's only one level of relativity (`./`), relative() would cut `./` making i.e. expected `./my/module` into `my/module`: it goes into imports like this then `import { some } from 'my/module'` which makes resolver think it's an alias or a node lib, whereas what we really need is `import { some } from './my/module'` to reflect that we import relatively to the same dir.

Reply:
Can we name this something like `ensureRelativeSignature` or something along those lines. This name doesn't really explain what the function does.

Reply:
`ensureRelativeForm` sounds good, I didn't come up with a better name. Renamed!